src/Views/sedimentview.html

```diff
@@ -102,7 +102,7 @@ <h5>Drainage area of clicked point: {{ vm.getDrainageArea() }}</h5>
                                     <td>{{gage.StartDate | date:'yyyy-MM-dd'}}</td>
                                     <td>{{gage.EndDate | date:'yyyy-MM-dd'}}</td>
                                     <td>
-                                        <a ng-class="{linkDisabled: gage.StationID == vm.selectedReferenceGage.StationID || !vm.checkCorrelationMatrix(gage) || !vm.checkPeriodOfRecord(gage) || !gage.HasFlowDurationStats || gage.stationType.code != 'GS'}" ng-click="vm.selectGage(gage)">
+                                        <a ng-class="{linkDisabled: gage.StationID == vm.selectedReferenceGage.StationID || !vm.checkPeriodOfRecord(gage) || gage.stationType.code != 'GS'}" ng-click="vm.selectGage(gage)">
```

Comment:
vm.checkCorrelationMatrix(gage) and gage.HasFlowDurationStats aren't relevant for this model